mf/StripStopwords/src/com/company/Main.java

```diff
@@ -5,14 +5,24 @@
 
 import java.io.IOException;
 
+/**
+ * This is an example of Stripper usage
+ */
 public class Main {
 
     public static void main(String[] args) {
-        // write your code here
+
         try {
-            String str = "a la mode";
-            Stripper s = new Stripper("//home//matteo//SoftwareDev//5030//data");
-            System.out.println(s.stripStopwords("fr", str));
```

Comment:
nice work on cleaning unnecessary comment!

Reply:
thank you